src/ui/ContextMenu/MenuItems.tsx

```diff
@@ -106,12 +107,11 @@ export default class MenuItems extends React.Component<MenuItemsProps, MenuItems
       return <></>;
 
     const { menuStyle } = this.state;
-    const { children, style, className = '' } = this.props;
-
+    const { children, style, className = '', testID = '' } = this.props;
```

Comment:
can we remove the default value of testID?
It seems make redundant attrobute like `data-testid=""`

Reply:
<img width="375" alt="image" src="https://github.com/sendbird/sendbird-uikit-react/assets/26326015/098364d5-2e49-44cb-863a-acef986dd04c">

Reply:
Due to unnecessary snapshot changes at present, it's difficult to view the diff.
Therefore, please request a review again when the changes are made.